src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import "./Comment.css";
+import { PostDropdown } from "../index";
+import { useDispatch } from "react-redux";
+import {
+  upvoteCommentOfPost,
+  downvoteCommentOfPost,
+} from "../../features/comments/commentsSlice";
+
+function Comment({
+  comment,
+  postId,
+  postUsername,
+  editCommentData,
+  setEditCommentData,
+}) {
+  const { _id, text, username, name, votes } = comment;
+
+  const [isCommentDropdownOpen, setIsCommentDropdownOpen] = useState(false);
+
+  const signInUser = localStorage.getItem("username");
+  const token = localStorage.getItem("token");
```

Comment:
You can take them directly in the function instead

Reply:
Yaah thats pending , I will do that